AcceptanceTests/runAcceptanceTests.sh

```diff
@@ -3,66 +3,90 @@
 echo "📴📴📴📴📴📴📴 Acceptance Testing has started 📴📴📴📴📴📴📴"
```

Comment:
I dont get what we gain from the renaming to temp, could you elaborate and help me understand?